### ISSUE TITLE
CQL issue with "Getting Started in 5 minutes" tutorial

### ISSUE THREAD

**Issue by adilfulara** [Impetus/kundera#544]
I followed you "Getting Started in 5 minutes" tutorial and was able to get it to work correctly.

I then tried to create the equivalent cal table for the same table. It seems that when i use "KunderaExamples" as the keyspace name, cal cannot use the specific keyspace. 

If i change the keyspace name to be all lower case, it does recognize the keyspace. 
Note that the query 

```
select * from system.schema_keyspaces;
```

still does report the keyspace to be present but i cannot issue the following command

```
use Kundera;
```

I think it would be better if the page was updated to use all lower cases so that a user can log into the cqlsh utility and list data added by the java sample code.

Also it would help if a comment is added to the persistence.xml to comment out the ehcache lines because that does create a problem.

Thank you,
Adil

**Comment by mevivs** [comment]
By default CQL3 case insensitive and assume it with lower case only. So in place of:

```
user Kundera;
```

Run

```
use "Kundera";
```

It should work for you. Also to use Kundera with CQL3 enabled, you can explicitly set it to CQL 3. https://github.com/impetus-opensource/Kundera/blob/22c58fbd8745544a92ae833f82a5e799413e1da8/src/kundera-cassandra/cassandra-core/src/test/java/com/impetus/client/cassandra/thrift/cql/PersonCassandraCQLTest.java#L18

should help you out.

-Vivek

**Comment by adilfulara** [comment]
Thank you @mevivs 

Any plans for Datastax Java drive support ?
Adil

**Comment by mevivs** [comment]
Issue #385 . Support available on current trunk will be releasing 2.11

-Vivek
